sdk/keyvault/azure-keyvault-keys/azure/keyvault/keys/_enums.py

```diff
@@ -39,6 +39,13 @@ class KeyOperation(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
     export = "export"
 
 
+class KeyRotationPolicyAction(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+    """The action that will be executed in a key rotation policy"""
+
+    ROTATE = "Rotate"  #: Rotate the key based on the key policy.
+    NOTIFY = "Notify"  #: Trigger Event Grid events. Not configurable in preview; defaults to 30 days before expiry.
```

Comment:
Consider removing the "Not configurable in preview;" part of the comment if this is something customers will see